samigo/samigo-app/src/webapp/js/authoring.js

```diff
@@ -787,4 +787,23 @@ function toggleCategories(checkbox) {
             categoryDiv.fadeOut();
         }
     }
-}
\ No newline at end of file
+}
+
+function expandAccordion(iframId){
+    $('div#jqueryui-accordion > .ui-accordion-content').show();
+    mySetMainFrameHeight(iframId);
```

Comment:
Is this frame height script still required now that we no longer have iFrames? Or does Lessons still require it for embedded quizzes?